litestar/types/helper_types.py

```diff
@@ -20,10 +18,7 @@
 
 T = TypeVar("T")
 
-__all__ = ("OptionalSequence", "SyncOrAsyncUnion", "AnyIOBackend", "StreamType", "MaybePartial")
-
-OptionalSequence: TypeAlias = Optional[Sequence[T]]
```

Comment:
I think that we can leave the type here, because users might import it, so technically this is a breaking change :)

Reply:
Good point.

Reply:
If we are just keeping this around for the sake of backward compatibility, should we deprecate it and remove in 3.0 @litestar-org/maintainers?

Reply:
Sounds good